src/commandDocs.json

```diff
@@ -14,8 +14,10 @@
   "start": {
     "description": "Starts a dev-server to build the Flex plugin locally",
     "flags": {
-      "name": "The name of the plugin(s) you would like to run",
-      "includeRemote": "Use this flag to include all remote plugins in your build"
+      "name": "The name of the plugin you would like to run",
+      "includeRemote": "Use this flag to include all remote plugins in your build",
+      "multiPluginsPilot": "Use this flag to support multi-plugins dev environment",
+      "nameMultiPlugins": "The name of the plugin you would like to run. You can provide multiple to run them all concurrently in the dev-environment."
```

Comment:
nit: this description is the only one with a full stop at the end. remove the last period so it's consistent with the rest?